packages/cosmic-proto/test/address-hooks.test.js

```diff
@@ -0,0 +1,214 @@
+import rawTest from '@endo/ses-ava/prepare-endo.js';
```

Comment:
Sometimes I wonder whether I should be using ses-ava. For my edification: what motivated you to use it here?